lib/secure_cookies.rb

```diff
@@ -0,0 +1,29 @@
+class SecureCookies
```

Comment:
Let's drop a comment here maybe explaining that the re-implements some of the SecureHeaders gem functionality that we want to keep?